src/net/socket.rs

```diff
@@ -77,7 +78,7 @@ impl Socket {
             // Now grab all the packets waiting to be sent and send them
             while let Ok(p) = self.packet_receiver.try_recv() {
                 if let Err(e) = self.send_to(p) {
-                    error!("There was an error sending packet: {:?}", e);
+                    println!("There was an error sending packet: {:?}", e);
```

Comment:
This shouldn't be a println!, especially as it is not in the testing code. println! goes right to stdout, and doesn't respect the logging facilities. This is especially important for users that might send logs off the box and won't be watching the stdout/stderr of one process.

Reply:
Seconded 👍